delivery_carrier_state/views/stock_picking.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_picking_withcarrier_out_form" model="ir.ui.view">
+        <field name="model">stock.picking</field>
```

Comment:
The "name" field is not required in views, it can be omitted.

Reply:
Ups!...
You got confused, it's the model :yum:

Reply:
Sorry i have seen it wrong.